docs/modules/servers/pages/distributed/configure/remote-delivery-error-handling.adoc

```diff
@@ -0,0 +1,107 @@
+= About RemoteDelivery error handling
+
+The advanced server mailQueue implemented by combining RabbitMQ for messaging and Cassandra for administrative operation
+do not support delays.
+
+Delays are an important feature for Mail Exchange servers, allowing to defer in time the retries, potentially letting the
+time for the remote server to recover. Furthermore, they enable implementation of advanced features like throttling and
+rate limiting of emails sent to a given domain.
+
+As such, the use of the distributed server as a Mail Exchange server is currently discouraged.
+
+However, for operators willing to inter-operate with a limited set of well-identified, trusted remote mail servers, such
```

Comment:
you also have to explain the limit of this solution, in particular that repositories are not meant to store transient data and so could be prone to tombstones.